components/ei-analytics/org.wso2.analytics.solutions.ei.analytics/widgets/EIAnalyticsMessageTable/src/EIAnalyticsMessageTable.jsx

```diff
@@ -23,7 +23,6 @@ import moment from 'moment';
 
 let TENANT_ID = '-1234';
 let MESSAGE_PAGE = "message";
-let PARAM_ID = "id";
```

Comment:
Seems like above variables are constants, please use `const` instead of `let`

Reply:
Fixed t b233d5335fc34cc9a03e24a4944cd39d0f6c3677